### PR TITLE
[REVIEW] Fix issue related to `na_values` input in `read_csv`

### PR DESCRIPTION
**Pull request by galipremsagar**
Fixes: #6678 , #6677 

This PR:

- [x] Adds `<NA>` to default list of `na_values`.
- [x] Fixes issue where `na_values` is a scalar and libcudf only expects list.

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

**Comment by galipremsagar** [comment]
rerun tests

File: python/cudf/cudf/utils/ioutils.py [hunk 1]
```diff
@@ -800,8 +800,13 @@
...
     line, the line will be ignored altogether.
-na_values : list, default None
-    Values to consider as invalid
+na_values : scalar, str, or list-like, optional
+    Additional strings to recognize as nulls.
+    By default the following values are interpreted as
+    nulls: ‘’, ‘#N/A’, ‘#N/A N/A’, ‘#NA’, ‘-1.#IND’,
+    ‘-1.#QNAN’, ‘-NaN’, ‘-nan’, ‘1.#IND’, ‘1.#QNAN’,
+    ‘<NA>’, ‘N/A’, ‘NA’, ‘NULL’, ‘NaN’, ‘n/a’, ‘nan’,
+    ‘null’.
```
Comment (kkraus14) [inline python/cudf/cudf/utils/ioutils.py:809]: These are non-default quote characters from copy pasting. Can you fix them?

**Review by kkraus14 — APPROVED** [review]

**Review by cwharris — APPROVED** [review]

File: python/cudf/cudf/utils/ioutils.py (hunk 1 above)
Comment (galipremsagar) [reply python/cudf/cudf/utils/ioutils.py]: Fixed it

**Review by rgsl888prabhu — APPROVED** [review]

**Review by vuule — APPROVED** [review]

**Pull request merged**